java/org/apache/coyote/http2/Stream.java

```diff
@@ -604,10 +611,23 @@ final void writeAck() throws IOException {
     final void writeEarlyHints() throws IOException {
         MimeHeaders headers = coyoteResponse.getMimeHeaders();
         String originalStatus = headers.getHeader(":status");
-        headers.setValue(":status").setString("103");
+        headers.setValue(":status").setString(String.valueOf(HttpServletResponse.SC_EARLY_HINTS));
```

Comment:
I am in favour of replacing 103 with the correct constant but that is this only part of the patch that I currently think should be retained. `Integer.toString()` would be better here though. There are also other places where 103 should be replaced.